src/presentation/webapp/sp-emergency-responses/EmergencyResponsesSyncHomePage.tsx

```diff
@@ -27,12 +26,46 @@ export const EmergencyResponsesSyncHomePage: React.FC<EmergencyResponsesSyncHome
     const rules = useSyncRulesList(emergencyType);
     const [isRunning, runSyncRule] = useSyncRulesExecuter({ emergencyType, logs });
 
+    const executeRules = React.useCallback(
+        (rules: SynchronizationRule[]) =>
+            _.orderBy(rules, rule => rule.code === "EFH_METADATA_ORGUNITS", "desc").forEach(rule => {
+                console.debug("Running: " + rule.name);
+                runSyncRule(rule);
+            }),
+        [runSyncRule]
+    );
+
+    const metadataRules = React.useMemo(() => {
+        const metadataRules = rules.filter(rule => rule.type === "metadata");
+        return { rules: rules, execute: () => executeRules(metadataRules) };
+    }, [rules, executeRules]);
+
+    const eventsRules = React.useMemo(() => {
+        const eventsRules = rules.filter(rule => rule.type === "events");
+        return { rules: rules, execute: () => executeRules(eventsRules) };
+    }, [rules, executeRules]);
```

Comment:
Not sure about these 2 structures. The "rules" property seems unused. And the execute would seem to be in fact a useCallback.

```typescript
const executeEventsRules = React.useCallback(() => {
    const eventsRules = rules.filter(rule => rule.type === "events");
    executeRules(eventsRules);
}, [rules, executeRules]);
```